app/models/spree/option_value_decorator.rb

```diff
@@ -1,10 +1,13 @@
 Spree::OptionValue.class_eval do
 
-  has_attached_file :image,
-    :styles        => { mini: '32x32>', normal: '128x128>' },
-    :default_style => SpreeVariantOptions::VariantConfig[:option_value_default_style],
-    :url           => SpreeVariantOptions::VariantConfig[:option_value_url],
-    :path          => SpreeVariantOptions::VariantConfig[:option_value_path]
+  has_attached_file(
+    :image,
+    styles: { mini: "32x32>", small: "64x64>", normal: "128x128>" },
+    default_style: SpreeVariantOptions::VariantConfig[:option_value_default_style],
```

Comment:
This is something that already existed, so it seems fair to ignore it for the purposes of this pull request.